liminal/kubernetes/secret_util.py

```diff
@@ -0,0 +1,120 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+import base64
+import logging
+import os
+import sys
+from pathlib import Path
+from time import sleep
+
+from kubernetes import client, config
+from kubernetes.client import V1Secret
+
+# noinspection PyBroadException
+try:
+    config.load_kube_config()
+except Exception:
+    msg = "Kubernetes is not running\n"
+    sys.stdout.write(f"INFO: {msg}")
+
+_LOG = logging.getLogger('volume_util')
+_LOCAL_VOLUMES = set()
+_kubernetes = client.CoreV1Api()
+
+
+def get_secret_configs(liminal_config, base_dir):
+    secrets_config = liminal_config.get('secrets', [])
+
+    for volume_config in secrets_config:
+        if 'secret' in volume_config and 'path' not in volume_config:
+            secret_path = f"{os.getcwd()}/credentials.txt"
```

Comment:
what is the purpose of this?
what happens when there are two secretes without a `path` configuration?

Reply:
I fixed the issue with multiple secrets